requirements-dev.txt

```diff
@@ -36,4 +36,8 @@ pymssql
 psycopg2
 connectorx>=0.2.6a4
 black
-flake8
\ No newline at end of file
+flake8
+# This should match the version in the CI. permalink from the time of this
+# commit:
+# https://github.com/modin-project/modin/blob/da6c9eb50f244f70f5e9be788d367103f567a6e5/.github/workflows/ci.yml#L55
+numpydoc==1.1.0
```

Comment:
Why didn't we add numpydoc in other dev env files?

Reply:
I'm sorry, I forgot about those. I have reopened the issue and created #4520 to resolve it. Please take a look.